form/src/main/java/org/dhis2/form/ui/provider/inputfield/FieldProvider.kt

```diff
@@ -266,24 +266,10 @@ internal fun FieldProvider(
 
             // "Remaining option sets" are in fun getLayoutForOptionSet
 
-            else -> { // Remove when all optionsets
-                AndroidViewBinding(
```

Comment:
have we implemented all option sets already to remove this else?

Reply:
If I'm not wrong, for rendering option sets only matrix and sequential are missing, which has just merged.

Reply:
great! I wasn't certain